package.json

```diff
@@ -73,5 +73,15 @@
     "buildConfig": {
       "minify": true
     }
-  }
+  },
+  "files": [
+    "bin",
+    "dist/*.js",
```

Comment:
I think you could replace all the `dist/...`-entries with just one `dist` entry.